modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/ObjectDefinitionExportImportTest.java

```diff
@@ -72,6 +75,112 @@ public void testExportImportObjectDefinition() throws Exception {
 			_baseObjectDefinitionJSONString,
 			_baseObjectDefinitionJSONString, null,
 			"TestObjectDefinitionJSON");
+
+		JSONObject actualJSONObject = JSONFactoryUtil.createJSONObject(
+			_baseObjectDefinitionJSONString
+		).put(
+			"externalReferenceCode", "TESTOBJECTDEFINITIONGERMAN"
```

Comment:
Can you change to TESTOBJECTDEFINITIONINGERMAN so it is on pair with the definition name?